### DISCUSSION TITLE
UmbrelOS 1.1.1

### DISCUSSION THREAD

**Discussion by aptonline** [tteck/Proxmox#2879] [Q&A]
A new version of UmbrelOS has been  released and with it a new way to install. It no longer uses a script to install on top of an existing OS (Ubuntu/Debian) but it's an OS in its own right. This would break the existing script?

https://github.com/getumbrel/umbrel/wiki/Install-umbrelOS-on-x86-systems

What would be the best way to install this on Proxmox?

**Comment by tteck** [comment]
The current script remains functional, but it's limited to version 0.5.4. While there's a chance I might develop a VM for umbrelOS 1.1 in the future, I can't make any promises.
